packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -999,5 +999,400 @@ void main() {
       );
     });
 
+    testWidgets('UiKitView accepts gestures', (WidgetTester tester) async {
+      final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+      final FakeIosPlatformViewsController viewsController = FakeIosPlatformViewsController();
+      viewsController.registerViewType('webview');
+
+      await tester.pumpWidget(
+        Align(
+          alignment: Alignment.topLeft,
+          child: SizedBox(
+            width: 200.0,
+            height: 100.0,
+            child: UiKitView(viewType: 'webview', layoutDirection: TextDirection.ltr,),
+          ),
+        ),
+      );
+
+      // First frame is before the platform view was created so the render object
+      // is not yet in the tree.
+      await tester.pump();
```

Comment:
I might be wrong but I think this one-frame-behind behavior isn't described in the docs for the classes is it?

Reply:
Added the following note on UiKitView:

```
/// Construction of UIViews is done asynchronously, before the UIView is ready this widget paints
/// nothing while maintaining the same layout constraints.
```